src/components/HorizontalRule/HorizontalRule.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+
+import HorizontalRule from "./HorizontalRule";
+
+const meta: Meta<typeof HorizontalRule> = {
+  title: "Components/Page Layout/HorizontalRule",
+  component: HorizontalRule,
+  decorators: [withDesign],
+  argTypes: {
+    className: { control: false },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof HorizontalRule>;
+
+/**
+ * Main Story for the HorizontalRule component. This must contains the `args`
+ *  and `parameters` properties in this object.
+ */
+export const WithControls: Story = {
+  args: {},
```

Comment:
Can you add `className: undefined` and also `align: undefined` so the args in Controls can be ordered.